cmd/mpi-operator.v1alpha2/app/server.go

```diff
@@ -42,6 +41,9 @@ import (
 	mpijobclientset "github.com/kubeflow/mpi-operator/pkg/client/clientset/versioned"
 	informers "github.com/kubeflow/mpi-operator/pkg/client/informers/externalversions"
 	"github.com/kubeflow/mpi-operator/pkg/version"
+	kubebatchclient "github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned"
```

Comment:
Can we place the deps in the second group? I think the third-party packages are always in the second group.